idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -310,6 +311,8 @@ def interpret_min_column_number_string(min_column_number_string,
                 min_column_number = interpret_min_column_number_string(
                     min_column_number_string, correct_number_of_output_columns,
                     try_number)
+
+                server.delete_job_tag(instance_iD, job_tag)
```

Comment:
nit: is there a reason you're capitalizing them as instance_iD or iP sometimes?

Reply:
yes. so they look less similar. Otherwise instance_id and instance_ip could easily be confused